ublox/src/ubx_packets.rs

```diff
@@ -168,45 +121,21 @@ pub struct NavPosLLH {
     pub height_msl: i32,
     pub horizontal_accuracy: u32,
     pub vertical_accuracy: u32,
-}*/
+}
 
-//ubx_meta!(NavPosLLH, 0x01, 0x02);
+ubx_meta!(NavPosLLH, 0x01, 0x02);
 
 impl From<&NavPosLLH> for Position {
     fn from(packet: &NavPosLLH) -> Self {
         Position {
-            lon: packet.get_lon() as f32 / 10_000_000.0,
-            lat: packet.get_lat() as f32 / 10_000_000.0,
-            alt: packet.get_height_msl() as f32 / 1000.0,
-        }
-    }
-}
-
-trait FooTrait {
-    fn foo() -> u32;
-}
-
-impl<T: FooTrait> From<&T> for Velocity {
-    fn from(packet: &T) -> Self {
-        Velocity {
-            speed: 0.0,
-            heading: 0.0,
+            lon: packet.lon as f32 / 10_000_000.0,
+            lat: packet.lat as f32 / 10_000_000.0,
+            alt: packet.height_msl as f32 / 1000.0,
         }
     }
 }
 
-#[ubx_packet]
```

Comment:
> I would like disscuss and merge this and then make another PR, that summarize our discussion about proc macro usage, syntax and so on things.

It may make sense to leave the `ubx_packet` macro usages in for this PR, if in the end we decide to go with something similar no sense in removing them now.

Reply:
Done